packages/react-docsite-components/src/components/PageHeader/PageHeader.tsx

```diff
@@ -39,15 +58,47 @@ const getStyles: IStyleFunction<IPageHeaderStyleProps, IPageHeaderStyles> = prop
 const getClassNames = classNamesFunction<IPageHeaderStyleProps, IPageHeaderStyles>();
 
 const PageHeaderBase: React.FunctionComponent<IPageHeaderProps> = props => {
-  const { pageTitle = 'Page title', pageSubTitle, theme } = props;
-  const styles = getClassNames(getStyles, { theme });
+  const { className, pageTitle = 'Page title', pageSubTitle, theme, versionSwitcherDefinition } = props;
+  const styles = getClassNames(getStyles, { className, theme });
+
+  const versionOptions: IContextualMenuItem[] | undefined =
+    versionSwitcherDefinition && versionSwitcherDefinition.versions
+      ? versionSwitcherDefinition.versions.map(version => ({
+          key: version,
+          text: version,
+          checked: version === versionSwitcherDefinition.currentVersion,
```

Comment:
I don't think the `checked` logic is working

Reply:
I think it's working, it just isn't doing anything with it, nor is it doing anything with it in the front page either. Should we just remove it since it's not doing anything either way?

Reply:
You mean the ActionButton isn't using it? If so let's just remove it.